app.js

```diff
@@ -4,26 +4,49 @@ const favicon = require('serve-favicon');
 const logger = require('morgan');
 const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+const passport = require('passport');
+const localStrategy = require('passport-local').Strategy;
 
-const routes = require('../routes/index');
-const users = require('../routes/users');
+
+const routes = require('./routes/index');
 
 const appRoot = process.env.APP_ROOT
 const app = express();
 
 // view engine setup
-app.set('views', path.join(__dirname, '../views'));
+app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'pug');
 
 app.set('env', process.env.PORT || '3000')
 if( process.env.NODE_ENV !== 'test' ) app.use( logger( 'dev' ))
-// app.use( cookieSession({
-//   name: 'session',
-//   keys: [[ process.env.SESSION_KEY ]]
-// }))
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(require('express-session')({
+   secret: 'keyboard cat',
+   resave: false,
+   saveUninitialized: false
+}));
+app.use(passport.initialize());
+app.use(passport.session());
+app.use(express.static(path.join(__dirname, 'public')));
+
+
+app.use('/', routes);
+
+// passport config
+const Account = require('./models/account');
+passport.use(new localStrategy(Account.authenticate()));
```

Comment:
Can we move the passport setup out into another module? (See floworky for an example)